LVT/Classes/Program.cs

```diff
@@ -1,19 +1,34 @@
 ﻿using LVT.LVT.Services;
 using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 
 namespace LVT
 {
     public class Program
     {
-
         static void Main(string[] args)
         {
             //this takes an absolute path and needs to be adapted for each computer depending on where the file lives that you want to open - but this allows you to pass different template trees.
             //see Tests for an alternative way that uses project resources instead - so tests can be run no matter on whose environment
-            StreamReader file = File.OpenText(@"C:\Users\beckerfs\Documents\Projects\LVT\Lean-Value-Tree-Visualisation\LVT\SingleBranchLVT.json");
+            //will no longer be an issue when we start the program from the command line with the file location as argument
+            //StreamReader file = File.OpenText(@"C:\Users\beckerfs\Documents\Projects\LVT\Lean-Value-Tree-Visualisation\LVT\SingleBranchLVT.json");
+            StreamReader file = File.OpenText(@"C:\Users\beckerfs\Documents\Projects\LVT\Lean-Value-Tree-Visualisation\LVT\TwoGoalsTwoBetsLVT.json");
             JsonParser Parser = new JsonParser();
             LeanValueTree newTree = Parser.ParseJsonLVTFromStream(file);
+            VisionPresenter VP = new VisionPresenter();
+            string OrgaChart = VP.VisualizeToString(newTree.Vision);
+            
+            //Console.WriteLine(OrgaChart);
+
+            // this needs to be moved to separate function
+
+            Directory.CreateDirectory("C:\\temp");
+            string content = Properties.Resources.TemplatePart1 + OrgaChart + Properties.Resources.TemplatePart2;
```

Comment:
TemplatePart1 and TemplatePart2 aren't very descriptive of what they are useful for

Reply:
They could be renamed to HTMLOrgChartTemplatePart1 and HTMLOrgChartTemplatePart2 - these, together with the string that's produced with all our 'presenters', is concatenated into the full text that makes up the .html file we're creating.  (Another bit of hardcoding that's really bad and that I don't like).